tasks/main.yml

```diff
@@ -4,3 +4,7 @@
 - name: shell | docker login
   shell: >
     docker login --username={{ docker_registry_username }} --password={{ docker_registry_password }} {{ docker_registry_url }}
+
+- name: shell | docker login to aws ecr
+  shell: >
+    $(aws ecr get-login --no-include-email --region ap-southeast-1)
```

Comment:
Wouldn't it be better to get a region from variable, that we set here `https://github.com/iPriceGroup/docker/blob/master/teamcity-server/docker-compose.yml#L77`